File: Trappist/src/Promact.Trappist.Web/wwwroot/app/conduct/test_bundle_model.ts
import { Test } from '../tests/tests.model';
import { TestAttendee } from './test_attendee.model';
import { TestQuestions } from './test_conduct.model';

export class TestBundleModel {
    test: Test;
    testAttendee: TestAttendee;
    testQuestions: TestQuestions[];
}

File: Trappist/src/Promact.Trappist.DomainModel/ApplicationClasses/TestConduct/TestBundleModel.cs
using Promact.Trappist.DomainModel.ApplicationClasses.Test;
using Promact.Trappist.DomainModel.Models.TestConduct;
using System.Collections.Generic;

namespace Promact.Trappist.DomainModel.ApplicationClasses.TestConduct
{
    public class TestBundleModelAC
    {
        public TestAC Test { get; set; }

        public ICollection<TestConductAC> TestQuestions { get; set; }

        public TestAttendees TestAttendee { get; set; }
    }
}